event_profiling.h

```diff
@@ -0,0 +1,175 @@
+#ifndef USE_EVENT_PROFILING
+#define USE_EVENT_PROFILING 0
+#endif
+
+#ifndef RUBY_EVENT_PROFILING_H
+#define RUBY_EVENT_PROFILING_H 1
+
+#if USE_EVENT_PROFILING
+
+#include "ruby/internal/config.h"
+
+#include "vm_core.h"
```

Comment:
why do you need vm_core.h?

Reply:
To use `GET_RACTOR()` 

https://github.com/ruby/ruby/blob/d77de609bea15703b8b63cb285c6dbe1fc032ad1/vm_core.h#L1777